src/components/Filters/index.tsx

```diff
@@ -1,63 +1,26 @@
 import { faFilter } from '@fortawesome/free-solid-svg-icons/faFilter';
-import { faTimes } from '@fortawesome/free-solid-svg-icons/faTimes';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { JSONSchema6 } from 'json-schema';
 import castArray from 'lodash/castArray';
 import cloneDeep from 'lodash/cloneDeep';
 import findIndex from 'lodash/findIndex';
 import includes from 'lodash/includes';
-import isEmpty from 'lodash/isEmpty';
-import isEqual from 'lodash/isEqual';
-import map from 'lodash/map';
 import reject from 'lodash/reject';
-import * as React from 'react';
+import React, { useState } from 'react';
```

Comment:
I think this might cause typing issues, so let's just keep it as `* as React` and use `React.useState`